lib/cdo/aws/cloudfront.rb

```diff
@@ -34,7 +34,7 @@ class CloudFront
       pegasus: {
         # NOTE: Keep this list in sync with the call to AWS::CloudFront.distribution_config in cloud_formation_stack.yml.erb.
         # CloudFormation stack should be refactored to reference this configuration in the future.
-        aliases: [CDO.pegasus_hostname, CDO.advocacy_hostname] + CDO.partners.map {|x| CDO.canonical_hostname("#{x}.code.org")},
+        aliases: [CDO.pegasus_hostname, CDO.pegasus_www_hostname, CDO.advocacy_hostname] + CDO.partners.map {|x| CDO.canonical_hostname("#{x}.code.org")},
```

Comment:
I think we don’t use the partners anymore (`CDO.partners` is empty on all of our environments, I think). Perhaps this is an opportunity to clean up that complexity?

Reply:
That's what I was thinking. Do you know where that gets populated?

Reply:
if it's empty everywhere (checking the sources listed in config/README) then lets get rid of it.